SetupScript.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+if [ $USER -ne "root"]
+	then
+	echo "Please run as root user"
+	exit 1
+fi
+#base script to fresh install systems onto base image for raspberry pi
+
+#check internet connection
+ping -q -c1 google.com > /dev/null
+
+if [ $? -ne 0 ]
+then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#configure displays
+(cd /opt/ && git clone https://github.com/UCSolarCarTeam/Epsilon-Raspberry.git)
+(cd /opt/Epsilon-Raspberry/primary/ && cp xorg.conf /etc/X11/ && cp config.txt /boot/)
```

Comment:
Neither of these calls require a subshell command. It'd be better to be explicit about where `xorg.conf` and `config.txt` are coming from.